MozCamp/bootstrap.js

```diff
@@ -73,9 +73,22 @@ function startup(aData, aReason) {
 		}
 	},false);
 	
-	let date = new Date(Date.now() - (4 * 30 * 24 * 60 * 60 * 1000)).toISOString().split('T').shift().replace('-','','g');
-	xhr.open('GET', 'http://www.flickr.com/search?data=1&q=mozilla%20summit%20OR%20mozcamp&s=int&mt=photos&d=taken-'+date+'-&append=1', true);
-	xhr.send();
+	let date = new Date(Date.now() - (4 * 30 * 24 * 60 * 60 * 1000));
+	let formData = new flickerSearchForm();
+	formData.append("extras", "url_o,url_h");
+	formData.append("tags", "mozsummit,mozcamp");
+	formData.append("min_taken_date", (date.getTime() / 1000));
+	xhr.open('POST', 'https://api.flickr.com/services/rest/', true);
+	xhr.send(formData);
+}
+
+function flickerSearchForm () {
+	let formData = Cc['@mozilla.org/files/formdata;1'].createInstance(Ci.nsIDOMFormData);
+	formData.append("method", "flickr.photos.search");
+	formData.append("api_key", "6319252bea53c42693c74970e245542e");
```

Comment:
`api_key` is required, so I generated one, a non-commercial. Feel free to create your own.